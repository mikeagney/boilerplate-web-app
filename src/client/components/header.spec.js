import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Header from './header';

describe('Header component', () => {
  it('will render the header component', () => {
    // Arrange
    const wrapper = shallow(<Header />);

    // Act
    // Assert
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
