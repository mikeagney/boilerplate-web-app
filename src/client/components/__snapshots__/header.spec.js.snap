// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component will render the header component 1`] = `
<div
  className="myapp-header"
>
  <h1>
    Header
  </h1>
</div>
`;
